Cookbook/Technical-Documents/ToolsAndServices.md

```diff
@@ -4,11 +4,11 @@
 
 | Service | What for | How to access |
 |---|---|---|
-| Slack | Chat with whole org. See channels of interest [here](NewHiresCheckList.md#slack). | You should have given access during your mini induction. Log in with your BabylonHealth email and Okta |
-| 1Password | Access shared passwords used by various services in the team | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) Specify in the ticket that access to iOS team vault is needed |
-| JIRA | Tickets of tasks and issues on your sprint | You should get an invitation email to join the BabylonPartners team after IT support ticket has been resolved [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) |
+| Slack | Chat with whole org. See channels of interest [here](NewHiresCheckList.md#slack). | You should have given access during your mini induction. Log in with your Babylon email and Okta |
+| 1Password | Access shared passwords used by various services in the team | [IT Support ticket];(https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) Specify in the ticket that access to iOS team vault is needed |
+| JIRA | Tickets of tasks and issues on your sprint | You should get an invitation email to join the BabylonPartners team after the [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) has been resolved |
 | Zeplin | Screen Designs | Ask `@Carlos` or `@adrian` on Slack for access to all the front-end projects |
```

Comment:
`@Carlos` left the company

Reply:
What about Adrian?

Reply:
Actually, anyone who already has access to the project can invite new members